lib/kitchen/directions/bake_chapter_glossary/v1.rb

```diff
@@ -12,8 +12,13 @@ def bake(chapter:, metadata_source:, append_to: nil, uuid_prefix: '')
 
       definitions = chapter.glossaries.search('dl').cut
       return if definitions.none?
-      definitions.sort_by! do |definition|
-        [definition.first('dt').text.downcase, definition.first('dd').text.downcase]
+
+      definitions.sort! do |def1, def2|
+        if def1.first('dt').text.downcase == def2.first('dt').text.downcase
+          I18n.sort_strings(def1.first('dd').text.downcase, def2.first('dd').text.downcase)
+        else
+          I18n.sort_strings(def1.first('dt').text.downcase, def2.first('dt').text.downcase)
+        end
```

Comment:
I see how you're working to make sorting use `I18n.sort_strings`.  When I look at this, I see a lot of repetition.  For example, 6 calls to `.text.downcase`, 6 calls to `.first(blah)`.  As you and the other developers get more senior, one goal is to recognize repetition like this and ask yourself (or your teammates) if there's a way to "DRY it up".  DRY = Don't Repeat Yourself.  

A maybe-too-aggressive way to DRY this up would be to override all String sorting by monkey patching the `String` class:

```ruby
class String
  def <=>(other)
    I18n.sort_strings(self, other)
  end
end
```

This may lead to unintended consequences elsewhere and will also be less efficient than the normal [C implementation of `<=>`](https://ruby-doc.org/core-2.6/String.html#method-i-3C-3D-3E).

Another more limited way would be to provide a helper method on `ElementBase`:

```ruby
class ElementBase
  def i18n_sortable_text
    I18nSortableText.new(text)
  end
```

Where you've added this new class to `kitchen/lib`:

```ruby
class I18nSortableText
  attr_reader :text

  def initialize(text)
    @text = text
  end

  def <=>(other)
    I18n.sort_strings(text.downcase, other.text.downcase)
  end
end
```

and then your `BakeChapterGlossary::V1` code becomes

```ruby
definitions.sort_by! do |definition|        
  [definition.first('dt').i18n_sortable_text, definition.first('dd').i18n_sortable_text]
end
```

A third way is to do what we did in `BakeIndex`, that is make a `Definition` class here that is given an element and can implement its own sorting method.